src/components/Modal/BecomeABuilderModal.tsx

```diff
@@ -198,6 +198,9 @@ const Steps: FC = () => {
         </Typography>
         .
       </li>
+      <li className="mt-4 list-none font-normal">
+        *You need to use a different address in case you stopped being a builder.
```

Comment:
@franciscotobar Where did you get this text from?

Reply:
Roger gave it to me:

![image](https://github.com/user-attachments/assets/e732dc1f-967c-4c2f-bcb8-3f7c015c9a6d)